### PR TITLE
feat: [CCM-7685]: Enhancements for AS dry run

### PR DESCRIPTION
**Pull request by rishik30**
##### Screenshots:
Screenshot 2022-05-20 at 3 49 13 PM
Screenshot 2022-05-20 at 3 52 26 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by rishik30** [comment]
retrigger jest

**Comment by rishik30** [comment]
retrigger jest

**Comment by rishik30** [comment]
retrigger jest

**Comment by rishik30** [comment]
retrigger jest

File: src/modules/75-ce/components/COGatewayList/COGatewayList.tsx [hunk 1]
```diff
@@ -790,32 +867,40 @@ const COGatewayList: React.FC = () => {
...
-      }
-      refetchServices()
+      onServiceDeletionSuccess(data)
     } else {
       showError(_defaultTo(data.data?.errors?.join(', '), ''))
     }
   }
 
+  const onServiceDeletionSuccess = (data: Service) => {
+    showSuccess(`Rule ${data.name} deleted successfully`)
```
Comment (rajarshee-harness) [inline src/modules/75-ce/components/COGatewayList/COGatewayList.tsx:896]: Can be moved to strings.

**Review by rajarshee-harness — APPROVED** [review]

File: src/modules/75-ce/components/COGatewayList/COGatewayList.tsx (hunk 1 above)
Comment (rishik30) [reply src/modules/75-ce/components/COGatewayList/COGatewayList.tsx:896]: will take up in the next ticket which will include further refactoring this code.

**Comment by rishik30** [comment]
retrigger jest

**Comment by rishik30** [comment]
retrigger jest

**Comment by rishik30** [comment]
retrigger jest

File: src/modules/75-ce/components/COGatewayList/COGatewayList.tsx (hunk 1 above)
Comment (jyotiarora-harness) [reply src/modules/75-ce/components/COGatewayList/COGatewayList.tsx:896]: Please try to make this changes in this branch only if possible

**Review by jyotiarora-harness — APPROVED** [review]

**Comment by rishik30** [comment]
retrigger jest

File: src/modules/75-ce/components/COGatewayList/COGatewayList.tsx (hunk 1 above)
Comment (rishik30) [reply src/modules/75-ce/components/COGatewayList/COGatewayList.tsx]: done

**Review by shalinlk — APPROVED** [review]

**Review by navaneeth — APPROVED** [review]

**Pull request merged**